Common/src/grid_movement_structure.cpp

```diff
@@ -4578,6 +5288,58 @@ void CSurfaceMovement::SetHicksHenne(CGeometry *boundary, CConfig *config, unsig
   
 }
 
+void CSurfaceMovement::SetSurface_Bump(CGeometry *boundary, CConfig *config, unsigned short iDV, bool ResetDef) {
```

Comment:
It looks like this just separates the existing capability to use a HH bump on an open surface from the HH bump routine for airfoils, correct? Or is there more to this change?